### DISCUSSION TITLE
Unsupported Pipeline - ... does not support depth to image.

### DISCUSSION THREAD

**Discussion by JuliusFroglet** [carson-katri/dream-textures#518] [General]
Hi,
I just wanted to try DT. However, it doesn't work and says;
"Unsupported Pipeline - The selected Pipeline does not support depth to image.

Is the Model meant? Because no model works.
Or what is the problem here?

Greatings

**Comment by carson-katri** [comment]
You cannot use the depth features with the Dream Studio version.

**Reply by JuliusFroglet** [reply]
I only have "Depth and Color" and "Depth" to choose from... 

What can I do about it?

![grafik](https://user-images.githubusercontent.com/108297135/212493396-7833c78a-5e07-4ea7-aa26-de1df1f9ea9f.png)

**Reply by JuliusFroglet** [reply]
In the Setup.md
its is mentioned, that I shall download a certain model. However, 
in the addon-preferences there is no such option.

![grafik](https://user-images.githubusercontent.com/108297135/212493675-2c446b60-cb64-4023-bce5-81bfbc637ddf.png)

**Reply by carson-katri** [reply]
As mentioned, it is not supported in the Dream Studio version.

**Reply by JuliusFroglet** [reply]
So Dreamtextures is currently not working with the cloud version, right?

**Reply by carson-katri** [reply]
You can use it for regular image generation in the image editor, just not the depth feature.

**Reply by JuliusFroglet** [reply]
Is there any possibility to do this, cludbased.
Or do I have to install it localy in any case?

**Reply by carson-katri** [reply]
You need local for the depth feature.

**Reply by JuliusFroglet** [reply]
Ok. Thanks for the response.
